### PR TITLE
Anqi performance- working and linked to eventlist

### LINKED ISSUE #62: Link Performance to Event

### LINKED ISSUE #65: Changing attendance & performance interaction
- PAC should be reading line by line with prompts

### PR DESCRIPTION
**Pull request by Anqi-nus**

### CODE REVIEW

File: src/main/java/seedu/ui/UI.java [hunk 1]
```diff
@@ -1,10 +1,13 @@
 package seedu.ui;
 
+import seedu.StudentList;
 import seedu.attendance.Attendance;
 import seedu.event.Event;
 import seedu.performance.Performance;
 import java.io.Serializable;
 import java.util.ArrayList;
+import java.util.HashMap;
```
Comment (benchan911) [inline src/main/java/seedu/ui/UI.java:9]: Where are you using HashMap?

Comment (benchan911) [reply src/main/java/seedu/ui/UI.java:9]: It doesn't seem used

Comment (Anqi-nus) [reply src/main/java/seedu/ui/UI.java:9]: ahhhhh i was trying hashmap for studentlist and forgot to delete

Comment (Anqi-nus) [reply src/main/java/seedu/ui/UI.java:9]: thanks for pointing it out!

**Review by lowjiayee — COMMENTED** [review]
Besides what @benchan911 has mentioned, all good.

**Pull request merged**